src/main/java/com/dougnoel/sentinel/steps/BaseSteps.java

```diff
@@ -1,8 +1,10 @@
 package com.dougnoel.sentinel.steps;
 
 import static com.dougnoel.sentinel.elements.ElementFunctions.getElement;
+
```

Comment:
I'm not sure what happened to this file, but there are way too many changes in it unrelated to what you were working on. Please revert it and only make the changes necessary for your code. This is too much to review.